app/services/search_service.rb

```diff
@@ -0,0 +1,16 @@
+class SearchService
+  def self.find_closest_station(location)
+    response = conn.get("/api/alt-fuel-stations/v1/nearest.json?location=#{location}")
+    body = parse_json(response)
+  end
+
+  def self.conn
+    Faraday.new(url: 'https://developer.nrel.gov/') do |faraday|
+      faraday.params['api_key'] = ENV['api_key']
```

Comment:
It can be helpful to use a little bit more specific variable name for each key. Something like `nrel_key` would be a little clearer to other devs joining the project.